src/content/en/2020/markup.md

```diff
@@ -441,7 +424,7 @@ Taking another look at element popularity, how likely is it to find a certain el
   <figcaption>{{ figure_link(caption="High probabilities of finding a given element in pages of the Web Almanac 2020 sample.", sheets_gid="184700688", sql_file="pages_element_count_by_device_and_element_type_present.sql") }}</figcaption>
 </figure>
 
-Standard elements are those that are or were part of the HTML specification. Which ones are you really rarely to find? In our sample, that would bring up the following:
+Standard elements are those that are or were part of the HTML specification, but which ones are rare to find? In our sample, that would bring up the following:
```

Comment:
I’m not sure this makes this clearer or gets it to read better. Do you feel strongly about this (why)? Suggest to leave this as it was.

Reply:
"Which ones are you really rarely to find" just does not read well as a native speaker. Maybe it's the alliteration of the r's.

```suggestion
Standard elements are those that are or were part of the HTML specification, but which ones are you rarely to find? In our sample, that would bring up the following:
```

Reply:
How about keeping this but rephrasing the second sentence then? “Which ones are rare to find?” That may address both our points?

Reply:
```suggestion
Standard elements are those that are or were part of the HTML specification. Which ones are rare to find? In our sample, that would bring up the following:
```